src/component/Card/Card.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import "./../Card/Card.css";
+import {
+	Image,
+	Button,
+	Rate,
+} from "antd";
+import { ShoppingCartOutlined } from "@ant-design/icons";
+const Card = ({ item1, onClick }) => {
+	return (
+		<>
+			{item1.map((x, id) => {
```

Comment:
use proper name item1 is not a proper name for props
also map callback variables is not with proper naming convention

use listItems instead of item1, or item and key instead of x and id